app/shared/video-player/common/ui/VideoScaffold.kt

```diff
@@ -75,7 +76,9 @@ fun VideoScaffold(
     danmakuHost: @Composable BoxScope.() -> Unit = {},
     gestureHost: @Composable BoxWithConstraintsScope.() -> Unit = {},
     floatingMessage: @Composable BoxScope.() -> Unit = {},
-    rhsBar: @Composable ColumnScope.() -> Unit = {},
+    rhsBarTop: @Composable ColumnScope.() -> Unit = {},
+    rhsBarCenter: @Composable ColumnScope.() -> Unit = {},
```

Comment:
实际上这个的意义已经变成了 gestureLock
```suggestion
    gestureLock: @Composable ColumnScope.() -> Unit = {},
```
可以把上面的 `rhsBarTop` 改成叫 `rhsButtons`?